apps/web/app/(use-page-wrapper)/403/page.tsx

```diff
@@ -27,4 +26,4 @@ async function Error403() {
   );
 }
 
-export default WithLayout({ ServerPage: Error403 });
+export default Error403;
```

Comment:
all the pages migrated in this PR do not have `getServerSideProps`, so they are lower-risk pages